src/praisonai-agents/tests/unit/test_degraded_delivery.py

```
@@ -0,0 +1,124 @@
+
+
+def test_report_is_frozen():
+    tbl = PresentationBlock.make_table(["a"], [["1"]])
+    _, report = adapt_presentation_with_report(
+        MessagePresentation([tbl]), PresentationLimits.telegram()
+    )
+    assert report is not None
+    try:
+        report.reasons = []  # type: ignore[misc]
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)